rubix-spi/src/main/java/com/qubole/rubix/spi/CacheConfig.java

```diff
@@ -112,6 +114,8 @@
   private static final String DEFAULT_METRICS_STATSD_HOST = "127.0.0.1"; // localhost
   private static final int DEFAULT_METRICS_STATSD_INTERVAL = 10000; // ms
   private static final int DEFAULT_METRICS_STATSD_PORT = 8125; // default StatsD port
+  private static final String DEFAULT_METRICS_GANGLIA_HOST = "127.0.0.1"; // localhost
+  private static final int DEFAULT_METRICS_GANGLIA_PORT = 8649; // default Ganglia port
   private static final String DEFAULT_METRICS_REPORTERS = "JMX";
```

Comment:
Can you make ganglia as default reporters along with jmx.

Reply:
will add Ganglia reporter as default